pkg/proxy/healthcheck/healthcheck_test.go

```diff
@@ -385,6 +388,10 @@ func TestHealthzServer(t *testing.T) {
 	testHealthzHandler(server, http.StatusOK, t)
 }
 
+func newFakeClient() clientset.Interface {
```

Comment:
This check will cause outages for cloud provider load balancer services that already have or are being told to move to the kube-proxy healthz port for default healthchecks as discussed [here](https://github.com/kubernetes/kubernetes/issues/14661#issuecomment-348332432) during apiserver outages.